manual/sphinx/user_docs/coordinates.rst

```diff
@@ -418,7 +418,15 @@ Differential operators in field-aligned coordinates
 These operators are valid for either `\left(x,y,z\right)` or
 `\left(x,\eta,z\right)` field-aligned coordinates defined
 above. Unless explicitly stated, in the sections that follow `y` will
-be used to indicate the parallel coordinate (`y` or `\eta`).
+be used to indicate the parallel coordinate (`y` or `\eta`). In a few
+places the sign of `B_\text{pol}` may appear, depending on whether `y`
+or `eta` is used for the parallel coordinate, so we define
+
+.. math::
+    \sigma_y = \begin{cases}
+        \sigma_{B\theta} & \text{if using }(x,y,z)
+        +1 & \text{if using }(x,\eta,z)
+    \end{cases}
```

Comment:
This hasn't rendered correctly, maybe

```suggestion
    \sigma_y = \begin{cases}
        \sigma_{B\theta} & \text{if using }(x,y,z) \\
        +1 & \text{if using }(x,\eta,z)
    \end{cases}
```

Reply:
Also, not completely relevant for this PR, but we use `\sigma_{B\theta}` and `B_pol` (and `B_tor`) -- worth changing one or the other to be consistent?

Reply:
It's true that there is some potential for confusion. I think it's good to keep `B_pol` and `B_tor` because these are different from `B_\theta` and `B_\phi` if the second are interpreted as contravariant components of the B vector (`B = B_\theta e^\theta + B_\phi e^\phi` so `B_pol = B_\theta / h_\theta`). 

Perhaps `\sigma_{B\theta}` should be `\sigma_{Bpol}` but not sure since they have the same sign. @johnomotani ?

Reply:
I agree it's good to keep `B_pol` and `B_tor`. I do think `\sigma_{B_pol}` would be better - we do use it for literally 'the sign of `B_pol`. I'll do a search-and-replace...